entries/SWC-137.md

```diff
@@ -0,0 +1,20 @@
+# Title
+Message call with hardcoded gas amount
+
+## Relationships
+[CWE-655: Improper Initialization](https://cwe.mitre.org/data/definitions/665.html)
+
+## Description
+
+The `transfer()` and `send()` functions forward a fixed amount of 2300 gas. Historically, it has often been recommended to use these functions for value transfers to guard against reentrancy attacks. However, the gas cost of EVM instructions may change significantly during hard forks which may break already deployed contract systems that make fixed assumptions about gas costs.  For example. [EIP 1884](https://eips.ethereum.org/EIPS/eip-1884) broke several existing smart contracts due to a cost increase of the SLOAD instruction. 
+
+## Remediation
+
+Avoid the use of `transfer()` and `send()` and do not otherwise specify a fixed amount of gas when performing calls. Use `.call.value(...)("")` instead. Use the checks-effects-interactions pattern to prevent reentrancy attacks. 
```

Comment:
To be consistent with https://github.com/SmartContractSecurity/SWC-registry/pull/240 should reentrancy locks be mentioned here?